### PR TITLE
Simplify the h1 and specify the behaviour for news details page.

### LINKED ISSUE #303: H1 elements are getting center aligned
image

### PR DESCRIPTION
**Pull request by bhellema**
Fix #303 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/new-homes/idaho/boise-metro/nampa/franklin-village-north/ashton
- After: https://issue-303--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/franklin-village-north/ashton

### CODE REVIEW

**Review by varunmitra — APPROVED** [review]

**Comment by codevoyage** [comment]
Reverting the change will cause side effects, such as misaligned headings on some pages. For instance, now the headings on About us and Contact Us pages are left-aligned instead of center-aligned:
https://issue-303--hubblehomes-com--aemsites.hlx.page/contact-us
https://issue-303--hubblehomes-com--aemsites.hlx.page/about-us

There needs to be a rule for these blue page headings specifically.

**Review by codevoyage — CHANGES_REQUESTED** [review]
Reverting the change will cause side effects, such as misaligned headings on some pages. For instance, now the headings on About us and Contact Us pages are left-aligned instead of center-aligned:
https://issue-303--hubblehomes-com--aemsites.hlx.page/contact-us
https://issue-303--hubblehomes-com--aemsites.hlx.page/about-us

There needs to be a rule for these blue page headings specifically.

**Comment by bhellema** [comment]
> Reverting the change will cause side effects, such as misaligned headings on some pages. For instance, now the headings on About us and Contact Us pages are left-aligned instead of center-aligned: https://issue-303--hubblehomes-com--aemsites.hlx.page/contact-us https://issue-303--hubblehomes-com--aemsites.hlx.page/about-us
> 
> There needs to be a rule for these blue page headings specifically.

I'll take a look at handing these other pages.

File: templates/about-us/about-us.css [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .about-us {
+  h1 {
+    text-align: center;
```
Comment (codevoyage) [inline templates/about-us/about-us.css:3]: Do we still need this if the global heading style is defined at the template level?

File: templates/news-detail/news-detail.css [hunk 2]
```diff
@@ -10,6 +10,11 @@
 
 .news-detail, .news-landing {
   .content-wrapper {
+    h1 {
+      text-align: center;
```
Comment (codevoyage) [inline templates/news-detail/news-detail.css:14]: Same for here, is this still needed?

File: templates/about-us/about-us.css (hunk 1 above)
Comment (bhellema) [reply templates/about-us/about-us.css]: I have removed this.  👍

File: templates/news-detail/news-detail.css (hunk 2 above)
Comment (bhellema) [reply templates/news-detail/news-detail.css]: I have removed this.  👍

**Review by codevoyage — APPROVED** [review]

**Pull request merged**